server/webapp/WEB-INF/rails.new/app/views/new_navigation_elements/_footer.html.erb

```diff
@@ -22,4 +22,9 @@
             </span>
       </div>
   </div>
+  <script type="application/javascript">
+    require(['models/shared/version_updater'], function(VersionUpdater) {
```

Comment:
I was looking at avoiding adding this script in each SPA, so was trying this option. Let me think bout and it we can look at the right way to do this.

```check_go_updates?``` needs to be added, will add it.